content/blog/easily-create-and-manage-aws-eks-kubernetes-clusters-with-pulumi/index.md

```diff
@@ -2,143 +2,337 @@
 title: Easily Create and Manage AWS EKS Kubernetes Clusters
 h1: "Easily Create and Manage AWS EKS Kubernetes Clusters with Pulumi"
 date: "2019-01-24"
-meta_desc: "Use Pulumi to easily deploy new clusters, manage AWS resources, and deploy Kubernetes apps. See how to provision an EKS cluster with a CLI single gesture."
+meta_desc: "Use Pulumi to easily deploy new clusters, manage AWS resources, and deploy Kubernetes apps. See how to provision an EKS cluster with a single CLI command."
 meta_image: "easy-eks.png"
-authors: ["luke-hoban"]
-tags: ["AWS","Kubernetes"]
+authors: 
+  - luke-hoban
+  - levi-blackstone
+tags:
+  - aws
+  - kubernetes
 ---
 
-
 Provisioning, managing, and monitoring a Kubernetes cluster is
-no easy task. AWS now offers EKS to ease a lot of that burden -- but
-it's still difficult to get up and running. Pulumi's infrastructure as
-code to the rescue! We can now provision an entire EKS cluster with a
-CLI single gesture, thanks to [the `@pulumi/eks`
-package](https://github.com/pulumi/pulumi-eks). Let's see how.
+no easy task. AWS now offers EKS to ease a lot of that burden -- but
+it's still difficult to get up and running. Pulumi's infrastructure as
+code to the rescue! We can now provision an entire EKS cluster with a
+single CLI command, thanks to [the `@pulumi/eks`
+package](https://github.com/pulumi/pulumi-eks). Let's see how.
 <!--more-->
 
+TODO: update this image?
 ![Pulumi making EKS Easy](./easy-eks.png)
 
+{{% notes type="info" %}}
+*Update (December 2020)*
+
+Our popular EKS library was developed
+back when Pulumi only supported TypeScript. Since that time, we've added support
+for additional languages, and have received frequent requests to bring EKS support
+to them as well. Today, we're excited to announce that EKS is available for Python,
+and support for .NET and Go is coming soon!
+{{% /notes %}}
+
 To get started, download the free and open source
-[Pulumi SDK]({{< relref "/docs/get-started/install" >}}). The SDK includes
-the CLI we'll be using below, and requires AWS credentials to access
+[Pulumi SDK]({{< relref "/docs/get-started/install" >}}). The SDK includes
+the CLI we'll be using below, and requires AWS credentials to access
 your AWS account and provision resources. If you already have the AWS
 CLI installed and configured, you're already all set to go. Otherwise,
 [check out the docs here]({{< relref "/docs/intro/cloud-providers/aws/setup" >}})
 to set things up.
 
-From here, there are two ways to proceed:
+From here, there are two ways to proceed:
 
 - Provision a new cluster with a single CLI command
 - Use infrastructure as code to manage clusters ... including Kubernetes resources themselves!
 
 ## Installing Amazon EKS with Pulumi
 
-With the Pulumi CLI installed and configured, a single command will
+TODO: is this still true now that we have multi-lang support?
```

Comment:
We could choose to make four copies of this in github.com/pulumi/apps - and then put a lang chooser here.  Keep the TypeScript one called `eks` to not break things.